### PR TITLE
fix(AndroidManifest.xml): Added background location permission

### PR DESCRIPTION
**Pull request by RamAkunuru**
Added background location permission in AndroidManifest.xml file.

### CODE REVIEW

**Review by nehabluedot — APPROVED** [review]

File: app/build.gradle [hunk 1]
```diff
@@ -36,6 +36,12 @@ android {
     }
 }
 
+android {
+    lintOptions {
+        abortOnError false
```
Comment (jsprincep) [inline app/build.gradle:42]: Why are we doing this?

File: app/src/main/res/values/appboy.xml [hunk 2]
```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <string name="com_appboy_api_key">3a0a1188-ee1d-4027-92c5-98c6033ad5f0</string>
+    <string name="com_appboy_api_key"></string>
```
Comment (jsprincep) [inline app/src/main/res/values/appboy.xml:3]: Do we need to scrub (filter-branch) the apiKeys from the repo history as well?

File: app/build.gradle (hunk 1 above)
Comment (RamAkunuru) [reply app/build.gradle:42]: That piece of code is suggested by Android Studio to fix the following issue:
Error: Version 28 (intended for Android Pie and below) is the last version of the legacy support library, so we recommend that you migrate to AndroidX libraries when using Android Q and moving forward.

File: app/src/main/res/values/appboy.xml (hunk 2 above)
Comment (RamAkunuru) [reply app/src/main/res/values/appboy.xml:3]: This is just an example integration and we don't want others to use our API keys.

Comment (jsprincep) [reply app/src/main/res/values/appboy.xml:3]: Ok, but the keys being visible in history is not a security issue?

File: app/build.gradle (hunk 1 above)
Comment (jsprincep) [reply app/build.gradle:42]: 👍 Maybe pop in a `\\ TODO remove this when we migrate to AndroidX`?

**Review by jsprincep — APPROVED** [review]

**Pull request merged**